src/Entity/News.php

```diff
@@ -21,11 +22,13 @@
 
     /**
      * @ORM\Column(type="string", length=1024)
+     * @Assert\NotBlank()
      */
```

Comment:
@daria-sieroshtan 
I suggest to add [Assert\Length](https://symfony.com/doc/current/reference/constraints/Length.html) (same for description)